### PR TITLE
Use WinRPM for Windows

### LINKED ISSUE #14: segfault when reading compressed MPS file
With https://github.com/mlubin/SimplexBenchmarks/blob/master/GenerateData/greenbea.gz:

```julia
julia> using GLPK

julia> lp = Prob()
Prob(Ptr{Void} @0x00000000045670d0)

julia> read_mps(lp, GLPK.MPS_FILE,"greenbea.gz")
Reading problem data from `greenbea.gz'...
Segmentation fault (core dumped)
```

Gdb trace:

```
#0  0x00007fffecde557c in crc32_little (len=<optimized out>, 
    buf=0x5810ac0 '*' <repeats 27 times>, "\n* SET UP THE INITIAL DATA *\n", '*' <repeats 27 times>, "\nNAME          GREENBEA", ' ' <repeats 58 times>, "\n*   Problem:\n*   ********\n*   An LP"..., 
    crc=<optimized out>) at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/zlib/crc32.c:259
#1  zlib_crc32 (crc=<optimized out>, buf=<optimized out>, len=len@entry=16384)
    at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/zlib/crc32.c:218
#2  0x00007fffecdeba4a in zlib_inflate (strm=strm@entry=0x5e27ff8, flush=flush@entry=0)
    at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/zlib/inflate.c:1232
#3  0x00007fffecde961a in gz_decomp (state=state@entry=0x5e27f80)
    at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/zlib/gzread.c:189
#4  0x00007fffecde97f6 in gz_fetch (state=state@entry=0x5e27f80)
    at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/zlib/gzread.c:245
#5  0x00007fffecde99d8 in zlib_gzread (file=0x5e27f80, buf=buf@entry=0x7fffffffcd8f, len=len@entry=1)
    at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/zlib/gzread.c:339
#6  0x00007fffecde9ae3 in zlib_gzgetc (file=<optimized out>)
    at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/zlib/gzread.c:398
#7  0x00007fffecd656db in z_fgetc (_fh=0x60096f0)
    at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/glpenv07.c:580
#8  _glp_lib_xfgetc (fp=<optimized out>)
    at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/glpenv07.c:214
#9  0x00007fffecda265f in read_char (csa=csa@entry=0x7fffffffd260)
    at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/glpmps.c:167
#10 0x00007fffecda2f43 in indicator (csa=0x7fffffffd260, name=1)
    at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/glpmps.c:214
#11 0x00007fffecda33ed in read_name (csa=0x7fffffffd260)
    at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/glpmps.c:395
#12 glp_read_mps (P=0x51c11a0, fmt=<optimized out>, parm=0x7fffffffcec0, fname=0x15cef20 "greenbea.gz")
    at /home/mlubin/.julia/GLPK/deps/src/glpk-4.52/src/glpmps.c:925
```

This used to work at some point a few months ago. If I uncompress the file, it seems to work fine.

### PR DESCRIPTION
**Pull request by blegat**
I currently get the following error when running `glp_test_1`, it is discussed [here](https://lists.gnu.org/archive/html/bug-glpk/2013-06/msg00012.html).
@tkelman How did you build the dll ?
```
Please submit a bug report with steps to reproduce this fault, and any error messages that follow (in their entirety). Thanks.
Exception: EXCEPTION_ACCESS_VIOLATION at 0x6a8271d0 -- glp_time at /home/abuild/rpmbuild/BUILD/glpk-4.61/src\env\time.c:61
while loading C:\Users\appveyor\.julia\v0.5\GLPK\test\glpk_tst_1.jl, in expression starting on line 297
gmtime at /home/abuild/rpmbuild/BUILD/glpk-4.61/src/usr/x86_64-w64-mingw32/sys-root/mingw/include\time.h:217 [inlined]
glp_time at /home/abuild/rpmbuild/BUILD/glpk-4.61/src\env\time.c:60
_glp_spx_primal at /home/abuild/rpmbuild/BUILD/glpk-4.61/src\simplex\spxprim.c:1125
```
Closes #14 #17

### CODE REVIEW

**Comment by tkelman** [comment]
Looking at https://github.com/JuliaOpt/GLPK.jl/pull/26 I think I just made a copy of existing binaries and put them somewhere more reliable than sourceforge, didn't build it myself.

File: deps/build.jl [hunk 1]
```diff
@@ -38,7 +38,14 @@ if is_apple()
...
 
 # Windows
-provides(Binaries, URI("https://bintray.com/artifact/download/tkelman/generic/win$glpkwinname.zip"),
-         glpkdep, unpacked_dir="$glpkwinname/w$(Sys.WORD_SIZE)", os = :Windows)
+@static if is_windows()
+    # GMP
+    using WinRPM
+    WinRPM.install("libgmp10", yes = true)
+    push!(WinRPM.sources, "https://cache.julialang.org/http://download.opensuse.org/repositories/home:/blegat:/branches:/windows:/mingw:/win32/openSUSE_13.2")
+    push!(WinRPM.sources, "https://cache.julialang.org/http://download.opensuse.org/repositories/home:/blegat:/branches:/windows:/mingw:/win64/openSUSE_13.2")
```
Comment (tkelman) [inline deps/build.jl]: better to use 42.2. or once this is working, this is a big enough library that may as well submit it to the official list

**Comment by blegat** [comment]
@tkelman I have added the 32bits and 64bits packages to the official projects on OBS and AppVeyor is passing now without needed to add my fork in WinRPM first.
Do you have any objection to merge this PR now ?

File: deps/build.jl [hunk 2]
```diff
@@ -38,7 +38,11 @@ if is_apple()
 end
 
 # Windows
-provides(Binaries, URI("https://bintray.com/artifact/download/tkelman/generic/win$glpkwinname.zip"),
-         glpkdep, unpacked_dir="$glpkwinname/w$(Sys.WORD_SIZE)", os = :Windows)
+@static if is_windows()
+    # GMP
+    using WinRPM
+    WinRPM.install("libgmp10", yes = true)
```
Comment (tkelman) [inline deps/build.jl]: this should be a dependency?

**Comment by tkelman** [comment]
I meant winrpm should handle the gmp dependency, probably doesn't need bindeps code for it

**Comment by mlubin** [comment]
Closed by #75

**Pull request closed without merging**